roles/openshift_facts/vars/main.yml

```diff
@@ -2,6 +2,7 @@
 required_packages:
   - iproute
   - python-dbus
+  - python-six
```

Comment:
If we have no other dependencies on six, I'd be happy to drop the metaclass and have one less dependency installed by openshift_facts.
Or do the PY2/PY3 metaclass dance manually, without six.

@detiber, thoughts?